assets/pages/regular-contributions/helpers/ajax.js

```diff
@@ -262,6 +263,10 @@ function handleStatus(
     response.json().then((statusResponse: StatusResponse) => {
       trackingURI = statusResponse.trackingUri;
 
+      if(statusResponse.guestAccountCreationToken) {
```

Comment:
Minor: currently we will dispatch this action every time we poll, which seems unnecessary.

It might be more efficient to do this as part of the success case? 

Perhaps we can have a helper function which dispatches the checkoutSuccess and (optionally) the guestAccountCreationToken?

Reply:
It isn't actually necessarily the `success` case that returns the token, it is when the `/contribute/recurring/create` is hit, and the status is usually pending at this point.

Also, it will only dispatch it once, as it is only this one call to the endpoint were the token is included. The other times this function is called, when actually getting the status of the payment, the token isn't returned

Reply:
Thanks for explaining 👍